### PR TITLE
Issue 1509

### PR DESCRIPTION
**Pull request by michael-grunder**
ZPOPMIN/ZPOPMAX BZPOPMIN/BZPOPMAX

### CODE REVIEW

**Comment by michael-grunder** [comment]
The errors appear to be igbinary/php5 related

File: redis_commands.c [hunk 1]
```diff
@@ -3052,14 +3061,6 @@ int redis_blpop_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
         "BLPOP", sizeof("BLPOP")-1, 2, 1, cmd, cmd_len, slot);
 }
 
-/* BRPOP */
```
Comment (yatsukhnenko) [inline redis_commands.c]: BLPOP also should be removed?

File: redis_commands.c [hunk 2]
```diff
@@ -3052,14 +3061,6 @@ int redis_blpop_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
...
-int redis_brpop_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
-                    char **cmd, int *cmd_len, short *slot, void **ctx)
-{
-    return gen_varkey_cmd(INTERNAL_FUNCTION_PARAM_PASSTHRU, redis_sock,
-        "BRPOP", sizeof("BRPOP")-1, 1, 1, cmd, cmd_len, slot);
```
Comment (yatsukhnenko) [inline redis_commands.c]: `min_argc` here is 1 but in `redis_varkey_timeout_cmd` you use 2. Where is th bug? :smile:

Comment (michael-grunder) [reply redis_commands.c]: Good catch I'll take a look.

File: redis_commands.c (hunk 1 above)
Comment (michael-grunder) [reply redis_commands.c]: Yes probably, will look.

File: redis_commands.c (hunk 2 above)
Comment (michael-grunder) [reply redis_commands.c]: I think the bug was in the old code.  All blocking pops seem to require at least one key and a timeout as far as I can tell.

https://redis.io/commands/BLPOP
https://redis.io/commands/bzpopmin

**Comment by jemmyphan** [comment]
will this be merged soon?

**Comment by michael-grunder** [comment]
Yes I will merge this weekend, and then reimplement it in the PHP7 only version.

**Pull request merged**